src/components/structures/MatrixChat.tsx

```diff
@@ -1638,7 +1638,7 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
             } else {
                 // otherwise check the server to see if there's a new one
                 try {
-                    newVersionInfo = await cli.getKeyBackupVersion();
+                    newVersionInfo = (await cli.getCrypto()?.getKeyBackupInfo()) || null;
```

Comment:
```suggestion
                    newVersionInfo = (await cli.getCrypto()?.getKeyBackupInfo()) ?? null;
```

Reply:
`cli.getCrypto()?.getKeyBackupInfo()` can return `undefined`, `null` or `KeyBackupInfo`. Why forcing the nullish coalescing ?

Reply:
nullish coalescing works for both `undefined` & `null` and is more strictly speaking accurate here, ultimately they do the same thing for the return types specified, but are more safe as they do not handle `"" | 0 | false` in the same manner so will trip type checks if the return type was to ever be changed to include `string | boolean | number`

Reply:
If we want to use it by default instead of `||`, we should specify it in the code style

Reply:
The code style doesn't state to use `||` either - one could infer that one should use `if` statements or ternary everywhere in the absence of such a rule

Reply:
I mean nullish coalescing is clearly a complement/improvement of the old/casual `||`.